src/content/docs/telemetry-data-platform/manage-data/query-limits.mdx

```diff
@@ -204,5 +204,55 @@ Attributes on `newrelic.resourceConsumption.impact`
 - `Impact` - A count of what is happening when resource has exceeded set limit, i.e dropped requests. 
 - `consumingAccountId` - The New Relic account where the resource is being consumed.
 
- 
+## Set alerts on resource metrics
+
+While building a dashboard to see all your limits is handy, being able to automate it is even better. You can set alerts on your limit metrics to provide updates on limits changes.
+
+<Callout variant="tip">
+Because we currently only have metrics on 1 minute time windows, setting TimeWindow = 1 minute, will cover them all. Eventually, we make more metrics available, you might want to set separate alerts for limits that are enforced by different time windows.
+</Callout>
+
+You can use the following NRQL queries to create alerts. Learn about creating alerts with [NRQL queries here](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#create).
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id=""
+    title="Limits faceted by LimitName and scoped by Timewindow"
+  >
+```
+From Metric select (rate(sum(newrelic.resourceConsumption.currentValue), 1 minute)/latest(newrelic.resourceConsumption.limitValue))*100 facet limitName
```

Comment:
do we somewhere explain the newrelic.resourceConsumption attribute? if they are all using this, maybe be worth explaining. And maybe it's something we add to data dictionary if it's common/important?